scripts/rnnlm/get_word_features.py

```diff
@@ -132,6 +142,12 @@ def read_features(features_file):
     unigram_probs = None
 feats = read_features(args.features_file)
 
+def treat_as_bos(word):
```

Comment:
This is super inefficient.  Figure out a better way to do this, e.g. create a set 'treat_as_bos_set' and do `if word in treat_as_bos_set:`.